chart/crds/vaultdynamicsecret-crd.yaml

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: MPL-2.0
 
+---
```

Comment:
Not sure if it matters, but the other chart crd yaml's don't have the `---` separator.

Reply:
good catch. I will update those in another PR.

Reply:
Fixed in #188